doc/index.rst

```diff
@@ -48,12 +48,12 @@ CCArchiver
 
 .. update the following links to point to your CSC (rather than the athexapod)
 .. image:: https://img.shields.io/badge/GitHub-dm_CCArchiver-green.svg
-    :target: https://github.com/lsst-ts/dm_CCArchiver
+    :target: https://github.com/lsst-dm/dm_CCArchiver
 .. image:: https://img.shields.io/badge/Jenkins-dm_CCArchiver-green.svg
     :target: https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/dm_CCArchiver/
 .. image:: https://img.shields.io/badge/Jira-dm_CCArchiver-green.svg
     :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+dm_CCArchiver
-.. image:: https://img.shields.io/badge/ts_xml-{cookiecutter.csc_name}}-green.svg
+.. image:: https://img.shields.io/badge/ts_xml-dm_CCArchiver-green.svg
     :target: https://ts-xml.lsst.io/sal_interfaces/CCArchiver.html
 
 .. TODO: Delete the note when the page becomes populated
```

Comment:
I think you can remove this warning now. If you prefer, you can make it a note that says something like, "This CSC documentation is still under development and is subject to change"

Reply:
I removed it.